server/src/main/java/io/openbytes/odi/application/ODIResourceService.java

```diff
@@ -0,0 +1,32 @@
+package io.openbytes.odi.application;
+
+import cn.hutool.core.util.StrUtil;
+import io.openbytes.odi.domain.ODIResource;
+import io.openbytes.odi.domain.storage.Storage;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.Collections;
+import java.util.List;
+
+@Slf4j(topic = "metricsLog")
+@Service
+public class ODIResourceService {
+
+    private final Storage storage;
+
+    public ODIResourceService(Storage storage) {
+        this.storage = storage;
+    }
+
+    public List<ODIResource> list(String datasetName, String marker, int maxKeys) {
```

Comment:
need some comments for parameters, maybe later?

Reply:
later would be good, I also need to update the response, add pageInfo in response.